features/supplier/opportunities.feature

```diff
@@ -75,3 +75,15 @@ Scenario: Checking all statuses returns all results
   And I check all 'status' checkboxes
   And I click the 'Filter' button
   Then I see that the stated number of results equals that result_count
+
+@withdrawn
+Scenario: See detail page for a withdrawn brief
+  Given I have a live digital outcomes and specialists framework
+    And I have a buyer
+    And I have a withdrawn digital-specialists brief
+  When I go to that brief page
```

Comment:
It seems like we're doing two different things for indentation.

1. Sometimes we align everything based on the first letter of the first word

```
  Given I am on the /g-cloud page
  When I have a random g-cloud lot from the API
  When I click that lot.name
  Then I am on the 'Search results' page
  And I see that lot.name breadcrumb
  And I see a search result
```

2. Sometimes we align things based on the last letter of the first word (so that the "I"s are aligned)

```
  Given 'Describe question and answer session' should not be ticked
   When I click 'Describe question and answer session'
    And I answer all summary questions
    And I click 'Return to overview'
```

In this file, it looks like we're aligning based on the first letter of the first word (the first example), so that's probably what you should do.